types/pepperlFuchs/wilsenSonicLevel/meta.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "WILSEN.sonic.level",
+  "version": "1.0.0",
+  "manufacturer": "Pepperl+Fuchs Vertrieb Deutschland GmbH",
```

Comment:
Taking a part of the companies name instead of the full name never made sense to me so I put the full name here. I changed it so it's still the full name, but shorter and more readable.